### ISSUE TITLE
setLookAt with transition enabled never resolves with off-center target

### ISSUE THREAD

**Issue by jiehui-gelement** [yomotsu/camera-controls#255]
## Describe the bug
When the target is off center, as what happens in the "click-to-set-orbit-point" sample linked below, setLookAt with enabledTransition true will never resolve.

 https://yomotsu.github.io/camera-controls/examples/click-to-set-orbit-point.html

## To Reproduce
1. go to https://yomotsu.github.io/camera-controls/examples/click-to-set-orbit-point.html
2. left click on the cube to change the orbit point
3. open console and run the following code: 
```
cameraControls.setLookAt(10, 10, 10, 0, 0, 0, true).then((a)=>{console.log('setLookAt done')});
```
4. notice how it never resolves

## Expected behavior
The promise returned by setLookAt should resolve when transition has ended

## Platform:

 - Device: Desktop
 - OS: Windows 10
 - Browser: Tested in Firefox & Chrome

**Comment by jiehui-gelement** [comment]
i think the issue is with the following code:
https://github.com/yomotsu/camera-controls/blob/f17306642de33d1868c1a36b84b31b671f35340d/src/CameraControls.ts#L1778

`deltaOffset` is not cloned, but is set to reference `_v3B`. `_v3B` is changed during subsequent update calculations, which causes value of deltaOffset to be wrong in the following line, which prevents the "rest" event from firing.
https://github.com/yomotsu/camera-controls/blob/f17306642de33d1868c1a36b84b31b671f35340d/src/CameraControls.ts#L1928

**Comment by yomotsu** [comment]
Thanks for your report and nice findings.
I'll take a look.
But sorry let me take sometime. I cought a COVID-19 according to my doctor...

**Comment by jiehui-gelement** [comment]
Thank you for the quick fix! Hope you are doing well, with covid.

Do you have any estimated date that the fix will be released?

**Comment by yomotsu** [comment]
Thanks! I almost recovered.
The fixed version v1.34.3 has been already available!

**Comment by jiehui-gelement** [comment]
the new version works.

thank you so much for the speedy fix & release!
